server/core/region.go

```diff
@@ -232,39 +229,17 @@ func (r *RegionInfo) GetStoreVoter(storeID uint64) *metapb.Peer {
 
 // GetStoreLearner returns the learner peer in specified store.
 func (r *RegionInfo) GetStoreLearner(storeID uint64) *metapb.Peer {
-	for _, peer := range r.Learners {
+	for _, peer := range r.learners {
 		if peer.GetStoreId() == storeID {
 			return peer
 		}
 	}
 	return nil
 }
 
-// RemoveStorePeer removes the peer in specified store for test use.
-func (r *RegionInfo) RemoveStorePeer(storeID uint64) {
-	var peers []*metapb.Peer
-	for _, peer := range r.GetPeers() {
-		if peer.GetStoreId() != storeID {
-			peers = append(peers, peer)
-		}
-	}
-	r.Peers = peers
-	classifyVoterAndLearner(r)
-}
-
-// AddPeer adds the peer in region info for test use.
-func (r *RegionInfo) AddPeer(peer *metapb.Peer) {
-	r.Peers = append(r.Peers, peer)
-	if peer.IsLearner {
-		r.Learners = append(r.Learners, peer)
-	} else {
-		r.Voters = append(r.Voters, peer)
-	}
-}
-
 // GetStoreIds returns a map indicate the region distributed.
 func (r *RegionInfo) GetStoreIds() map[uint64]struct{} {
-	peers := r.GetPeers()
+	peers := r.meta.GetPeers()
```

Comment:
It seems we can use `GetPeers` directly?

Reply:
I want to change the export function `GetPeers` without getting the pointer of the origin peers in the next PR. now we got the peer can also change the read-only  `RegionInfo`. So, in the inner, I still use the method of meta.